tools/serverpod_cli/lib/src/analyzer/entities/entity_analyzer.dart

```diff
@@ -51,32 +51,20 @@ class SerializableEntityAnalyzer {
   }) async {
     var classDefinitions = <SerializableEntityDefinition>[];
 
-    // Get list of all files in protocol source directory.
-    var sourceDir = Directory(p.joinAll(config.protocolSourcePathParts));
-    var sourceFileList = await sourceDir.list(recursive: true).toList();
-    sourceFileList.sort((a, b) => a.path.compareTo(b.path));
+    var protocols =
+        await ProtocolHelper.loadProjectYamlProtocolsFromDisk(config);
 
-    var sourceDirPartsLength = p.split(sourceDir.path).length;
-
-    for (var entity in sourceFileList) {
-      if (entity is! File || !entity.path.endsWith('.yaml')) {
-        if (verbose) print('  - skipping file: ${entity.path}');
-        continue;
-      }
-      var subDirectoryParts =
-          p.split(p.dirname(entity.path)).skip(sourceDirPartsLength).toList();
-
-      // Process a file.
-      if (verbose) print('  - processing file: ${entity.path}');
```

Comment:
Question: When this logic was moved to the `protocol_helper.dart` class it looks like we lost a lot of information prints. Was that intentional?